proc/proc.go

```diff
@@ -118,10 +121,12 @@ func GetKernelModules(modulesPath string,
 		if nFields < 6 {
 			return nil, fmt.Errorf("unexpected line in modules: '%s'", line)
 		}
-		if address == 0 {
-			return nil, fmt.Errorf(
-				"addresses from modules is zero - "+
-					"check process permissions: '%s'", line)
+
+		if address != 0 {
+			atLeastOneValidAddress = true
+		} else {
+			lastInvalidAddressLine = line
+			continue
 		}
```

Comment:
```suggestion
		if address == 0 {
			lastInvalidAddressLine = line
			continue
		}
		atLeastOneValidAddress = true
```
Since the block has `continue`, no need for the else block.